src/libtrx/game/test_replay.c

```diff
@@ -319,7 +318,7 @@ static bool M_ParseBindController(
 
 static bool M_ParseArgs(const char *const line, M_PARSE_CTX *const ctx)
 {
-    if (strncmp(line, "# args", 6) != 0) {
+    if (strncmp(line, "args", 6) != 0) {
```

Comment:
I think 6 needs to become 4, replaying currently gives e.g. `WRN | 2025-07-25 18:09:01.132 [libtrx/game/test_replay.c:400:M_ReadHeaders] Unknown line: args "-l" "data/level5.phd"`.

There is a 6 also on line 326.

Reply:
Thanks – fixed!